src/layouts/sidebar-sidecar/utils/_local_platform-docs-mdx/components/enterprise-alert/index.tsx

```diff
@@ -16,34 +16,50 @@ import { isProductSlug, productSlugsToNames } from 'lib/products'
 import s from './enterprise-alert.module.css'
 
 interface EnterpriseAlertProps {
-	productSlug: ProductSlug
+	productSlug: ProductSlug | 'vault-secrets' | 'vault-radar'
 	inline?: boolean
 	className?: string
 	children?: ReactNode
+	badgeText?: string
 }
 
 export function EnterpriseAlert({
 	productSlug,
 	inline,
 	className,
 	children,
+	badgeText,
 }: EnterpriseAlertProps) {
 	// This ensures we aren't producing invalid HTML when rendering inline alerts within MDX. When used inline, we might end up nesting a div inside of a p. This is invalid as p cannot contain block-level elements (ref: https://www.w3.org/TR/html401/struct/text.html#h-9.3.1).
 	const Element = inline ? 'span' : 'div'
-	const isValidProduct = isProductSlug(productSlug)
+	const isValidProduct =
+		isProductSlug(productSlug) ||
+		productSlug === 'vault-radar' ||
+		productSlug === 'vault-secrets'
 
 	if (productSlug === 'hcp') {
 		return null
 	}
 
+	const productNames = {
+		...productSlugsToNames,
+		'vault-secrets': 'Vault Secrets',
+		'vault-radar': 'Vault Radar',
+	}
+
 	if (!isValidProduct) {
 		throw new Error(
 			`[EnterpriseAlert]: Invalid product option passed. Expected one of ${Object.keys(
-				productSlugsToNames
+				productNames
 			).filter((slug) => slug !== 'hcp')}`
 		)
 	}
 
+	const href =
+		productSlug === 'vault-radar' || productSlug === 'vault-secrets'
+			? `https://www.hashicorp.com/products/vault/hcp-${productSlug}`
+			: `https://www.hashicorp.com/products/${productSlug}`
+
```

Comment:
Since these pages link to the vault pricing page, can we axe this and change `href` back to `href={`https://www.hashicorp.com/products/${productSlug}`}`?

Reply:
As default, the href is the product page on the www site not the pricing page (example of this [here](https://developer.hashicorp.com/vault/docs/sync)). Since `https://www.hashicorp.com/products/hcp-vault-secrets` and `https://www.hashicorp.com/products/hcp-vault-radar` are not valid hashicorp.com pages, we need this so it doesn't redirect to a 404 page. A custom url can be passed in via the children if the vault pricing page is preferred.

Reply:
Ah, my mistake. I assumed the alert on [this page](https://developer.hashicorp.com/vault/docs/import) was the default!